typings/index.d.ts

```diff
@@ -0,0 +1,33 @@
+type CDeclaration = {
+  name: string;
+  type: 'char' | 'union' | 'struct' | string;
```

Comment:
should `string` be quoted here?

Reply:
No, the strings in this case are the other type names defined in the header (e.g. `C_WA2_F1A`)